tests/Helpers/FileHelper.php

```diff
@@ -8,4 +8,20 @@ public static function absolutePath($fileName, $diskName)
     {
         return config('filesystems.disks.' . $diskName . '.root') . DIRECTORY_SEPARATOR . $fileName;
     }
+
+    public static function recursiveDelete($fileName)
```

Comment:
I would prefer this helper to be within the test, so I won't be used outside of the tests and end up in some user-land code.

Reply:
The original `absolutePath` function is also in the `Tests\Helpers` namespace and also only used for tests, I followed the original idea for the test helpers here.

If this function is used within user-land code this would mean that a user include the class from the `Test\Helpers` namespace which is not in the composer package.

Reply:
Sorry you are right, it already is!